sourcegraph/sourcegraph
# Disable scheduled workflows in forks

I am getting about 100 messages about failed jobs after forking this repo. This PR should fix that.

## Test plan

For repo, wait a day and notice there are no these headers in your mailbox.

* Run failed: PR Preview - main
* Run failed: Progress bot - main
* Run failed: Tracking Issue Syncer - main
* Run failed: buildchecker - main
* Run failed: Resources Report - main
